bionic/protocols.py

```diff
@@ -197,6 +198,29 @@ def __repr__(self):
         return f"{self.__class__.__name__}(...)"
 
 
+class JsonProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values can be serialized using the
+    ``json`` library.
+    """
+
+    def get_fixed_file_extension(self):
+        return "json"
+
+    def validate(self, value):
+        # We don't count sequence types and dict as valid values. Otherwise, we
+        # will have to validate each entries in the sequences and dicts.
+        assert isinstance(value, (int, float, str, bool))
```

Comment:
Should we support `None`?

Reply:
Good point, I think we should support None.